secboot/openssl.cnf

```diff
@@ -0,0 +1,15 @@
+openssl_conf = openssl_init
```

Comment:
Is this file a leftover?

Reply:
yes, good catch. Now we are deploying OpenSSL config from flake.